src/test/java/com/ericsson/ei/subscriptionhandler/test/SubscriptionHandlerTest.java

```diff
@@ -52,36 +69,31 @@
 import org.springframework.util.LinkedMultiValueMap;
 import org.springframework.util.MultiValueMap;
 
-import javax.annotation.PostConstruct;
-import java.io.File;
-import java.io.IOException;
-import java.util.HashSet;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Set;
-
-import static org.junit.Assert.*;
-import static org.mockito.Mockito.*;
-import static org.powermock.reflect.Whitebox.invokeMethod;
+import de.flapdoodle.embed.mongo.distribution.Version;
+import de.flapdoodle.embed.mongo.tests.MongodForTestsFactory;
 
 @RunWith(SpringJUnit4ClassRunner.class)
-@SpringBootTest(classes = {App.class})
+@SpringBootTest(classes = { App.class })
 @AutoConfigureMockMvc
 public class SubscriptionHandlerTest {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(SubscriptionHandlerTest.class);
     private static final String aggregatedPath = "src/test/resources/AggregatedObject.json";
+    private static final String aggregatedInternalPath = "src/test/resources/AggregatedDocumentInternalCompositionLatest.json";
```

Comment:
private static final variables should be UPPER CASE, I would not complain here if all was lower, or all upper, but here is a mix match where some are upper some are lower.